src/modules/dashboard/components/ActionsPage/ActionsPageFeedItem/ActionsPageFeedItem.css

```diff
@@ -0,0 +1,36 @@
+
+.main {
```

Comment:
I think we need a bigger margin between the feed items. I see something like `30` in the Figma file

![Screenshot from 2020-12-02 16-52-54](https://user-images.githubusercontent.com/1193222/100888512-e0825300-34be-11eb-9397-79e72d55cc7d.png)

Reply:
<img width="350" alt="Screenshot 2020-12-02 at 17 10 02" src="https://user-images.githubusercontent.com/13069555/100898613-3c4ddb80-34c1-11eb-9da6-a36c8191263e.png">
Depends on which design we're looking for 🤦  23-16px of padding, it means something like 7-8px of margin, i can add it

Reply:
Yes, I know, the Figma file doesn't really give proper specs in this regard. 

I just like bigger margins visually :P

I think you can leave them as they currently are.